src/vs/workbench/contrib/tasks/test/browser/taskTerminalStatus.test.ts

```diff
@@ -53,10 +60,12 @@ suite('Task Terminal Status', () => {
 	let testTerminal: ITerminalInstance;
 	let testTask: Task;
 	let problemCollector: AbstractProblemCollector;
+	let audioCueService: TestAudioCueService;
 	setup(() => {
 		instantiationService = new TestInstantiationService();
 		taskService = new TestTaskService();
-		taskTerminalStatus = instantiationService.createInstance(TaskTerminalStatus, taskService);
+		audioCueService = new TestAudioCueService();
+		taskTerminalStatus = new TaskTerminalStatus(taskService as any, audioCueService as any);
```

Comment:
way to avoid `any` here? @tyriar

Reply:
You could stub the services and create it with `instantiationService`, like what is done here:

https://github.com/microsoft/vscode/blob/ea7afaee0a8814e329492106e485644f40a1ed45/src/vs/workbench/contrib/terminal/test/browser/links/terminalLinkOpeners.test.ts#L90-L108

Reply:
can't do that w the taskService so need to use any because `triggerStateChange` doesn't exist on the task service

Reply:
You should be able to do it with TaskService as well, just keep your taskService reference around when you need triggerStateChange:

```ts
instantiationService = new TestInstantiationService();
taskService = new TestTaskService();
audioCueService = new TestAudioCueService();
instantiationService.stub(ITaskService, taskService);
instantiationService.stub(IAudioCueService, audioCueService);
taskTerminalStatus = instantiationService.createInstance(TaskTerminalStatus);
```